server/dash/dashboards/neighborhood.py

```diff
@@ -41,13 +32,37 @@ def populate_options():
     return values
 
 
+layout = html.Div([
+    html.H1(title),
+    dcc.Dropdown(
+        id='council_list',
+        clearable=False,
+        value="Arleta",
+        placeholder="Select a neighborhood",
+        options=populate_options()
+    ),
+    dcc.Graph(
+        id='ncAvgCompLineChart',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    ),
+    dcc.Graph(
+        id='ncNumReqTypeLineChart',
+        figure=fig,
+        config=CONFIG_OPTIONS
+    )
+])
+
+
+
+
+
 # Define callback to update graph
-@app.callback(
-    Output('graph1', 'figure'),
+@callback(
+    Output('ncAvgCompLineChart', 'figure'),
```

Comment:
Python uses `snake_case`, so let's use that instead of `camelCase`. i.e., `nc_avg_comp_line_chart`.